sdk/include/crt/mingw32/intrin.h

```diff
@@ -31,12 +31,11 @@
 #ifndef RC_INVOKED
 
 #ifdef __clang__
-#define __ATTRIBUTE_ARTIFICIAL
+#define __INTRIN_INLINE extern inline __attribute__((__always_inline__,__gnu_inline__))
 #else
 #define __ATTRIBUTE_ARTIFICIAL __attribute__((artificial))
-#endif
-
 #define __INTRIN_INLINE extern __inline__ __attribute__((__always_inline__,__gnu_inline__)) __ATTRIBUTE_ARTIFICIAL
+#endif
```

Comment:
Those are some ugly defines.
lgtm

Reply:
Basically they mean "Inline this, I'm serious, do it!"

Reply:
__forceinline

Reply:
`__forceinline` doesn't even force inlining. It'll still be out-of-line in debug mode ;)